Tests/MeiliSearchUnitTests/SystemTests.swift

```diff
@@ -46,6 +46,50 @@ class SystemTests: XCTestCase {
 
     }
 
+    func testIsHealthy() {
+
+        //Prepare the mock server
+
+        let jsonString = """
+        {
+            "status": "available"
+        }
+        """
+
+        session.pushData(jsonString, code: 200)
+
+        // Start the test with the mocked server
+
+        let expectation = XCTestExpectation(description: "Check server health")
+
+        self.client.isHealthy { result in
+            XCTAssertEqual(result, true)
+            expectation.fulfill()
+        }
+
+        self.wait(for: [expectation], timeout: 1.0)
+
+    }
+
+    func testIsHealthyFail() {
+
+        //Prepare the mock server
+
+        session.pushData("", code: 400)
+
+        // Start the test with the mocked server
+
+        let expectation = XCTestExpectation(description: "Check server health")
+
+        self.client.isHealthy { result in
+            XCTAssertEqual(result, false)
+            expectation.fulfill()
+        }
```

Comment:
I see you'r not catching a possible error: 

Like the following example
```swift
     switch result {
                case .success:
                    expectation.fulfill()
                case .failure(let error):
                    print(error)
                    XCTFail()
                }
            }
```

What happens when `testIsHealthyFail` fails? Is the output correct?

Reply:
You right I fix it!